src/psyclone/tests/psyir/symbols/symboltable_test.py

```diff
@@ -378,8 +378,42 @@ def test_imported_symbols():
     assert ("The 'my_mod' entry in this SymbolTable is not the supplied "
             "ContainerSymbol" in str(err.value))
 
+def test_remove_genericsymbols():
```

Comment:
There are pylint errors about the number of blank lines before some of the new tests in this file.

Reply:
Done (I think)